apollo-android-support/src/main/java/com/apollographql/apollo/ApolloAndroidLogger.kt

```diff
@@ -0,0 +1,23 @@
+package com.apollographql.apollo
+
+import android.util.Log
+import com.apollographql.apollo.Logger
+
+/**
+ * This is an Android wrapper around [Logger] that will take any messages Apollo wants
+ * to print and log them in the Android logcat.
+ */
+class ApolloAndroidLogger : Logger {
```

Comment:
It's worth thinking about what this should mean for Android devs who don't want these logs in production, and whether Apollo wants to be opinionated about this. Perhaps we can provide a solution to turn them off, or just rely on devs to make sure they don't include this logger in production.

If we want to do that, maybe we can add some additional notes inside the documentation here.

Let me know what y'all think!

Reply:
>  Perhaps we can provide a solution to turn them off, or just rely on devs to make sure they don't include this logger in production.

I'd prefer that decision to be left to the caller. The definition of what "production" is might vary depending the teams/products so I think it's better if Apollo Android doesn't know anything about this. Also, not sending log is as easy as not setting a logger so it should be reasonnably doable at the callsite

Reply:
Agreed! So let me know if you think that warrants any additional documentation, but I don't think so - we should rely on the callers to be aware of this concept. :)